src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerCFTTest.java

```diff
@@ -459,7 +459,7 @@ public void should_return_a_200_with_task_description_property() {
             .body("task.case_type_id", notNullValue())
             .body("task.case_category", notNullValue())
             .body("task.case_name", notNullValue())
-            .body("task.auto_assigned", notNullValue())
+            .body("task.auto_assigned", equalTo(true))
```

Comment:
if this FT is covering "If auto-assignable is true + authorisation is empty = assign task" shouldn't the **task.task_state = "assigned"**